src/writeckpt.cpp

```diff
@@ -190,14 +184,7 @@ mtcp_writememoryareas(int fd)
     // will invoke mmap if the JAlloc arena is full. Similarly, for STL objects
     // such as vector and string.
 
```

Comment:
And I have a similar worry about removing this comment, and not documenting our consistency/design requirements between writeckpt.cpp and mtcp_restart.c.